patterns/newtype.md

```diff
@@ -1,5 +1,9 @@
 # Newtype
 
+Rust has strong static types. This can be very different than what you are used to if you are coming from a loosely-typed language. Don't worry, though. Once you get used to them, you'll find the types actually make your life easier. Why? Because you are making implicit assumptions explicit.
+
+A really convenient application of the Rust type system is the Newtype pattern.
```

Comment:
How is Newtype pattern convenient? I think we should use a different word other than convenient or maybe we should explain why is it convenient. Or maybe we could just

> An application of the Rust type system is the Newtype pattern.

Reply:
> What if in some cases we want to have a type to behave similarly but still different to another type?

I like this :)